projects/ffmpeg/build.sh

```diff
@@ -203,12 +206,11 @@ rm $FFMPEG_DEPS_PATH/lib/*.so.*
 
 # Build ffmpeg.
 cd /ffmpeg
-FFMPEG_BUILD_ARGS=''
 
 PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
-        --cc=$CC --cxx=$CXX --ld="$CXX $LDFLAGS -std=c++11" \
-        --extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
-        --extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
+        --cc=$CC --cxx=$CXX --ld="$CXX" \
+        --extra-cflags="-I$FFMPEG_DEPS_PATH/include $FFMPEG_CFLAGS" \
+        --extra-ldflags="-L$FFMPEG_DEPS_PATH/lib $FFMPEG_CFLAGS -std=c++11" \
```

Comment:
Maybe LDFLAGS, not FFMPEG_CFLAGS?

Reply:
in FFMPEG_CFLAGS there is ubsan flag that may be useful for linker. I have changed LDFLAGS name to CFLAGS so now it is more understandable